docs/docs-content/integrations/k3s.md

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_label: 'K3s'
+title: 'K3s'
+description: 'K3s pack in Palette Edge'
+hide_table_of_contents: true
+type: "integration"
+category: ['kubernetes', 'amd64', 'fips']
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: 'https://registry.spectrocloud.com/v1/edge-k3s/blobs/sha256:5b7f8a80330d6938056848905238efbbd085e2f0f86fd03d7be10ccdabf791c7?type=image/png'
+tags: ["packs", "kubernetes", "k3s", "edge"]
+---
+
+K3s is a lightweight distribution of Kubernetes, specifically designed for resource-constrained environments and IoT applications. Unlike standard Kubernetes with multiple binaries for its various components, K3s is packaged as a single binary with a lightweight storage backend and minimal external dependencies. For more information about K3s, refer to [K3s documentation](https://docs.k3s.io/). 
+
+## Versions Supported
+
+<Tabs queryString="versions">
```

Comment:
Looking at the Packs reporter, I see active versions in prod. We also need to specify these versions and created a tab for deprecated stating what is the cutoff.

![CleanShot 2023-11-15 at 09 48 54](https://github.com/spectrocloud/librarium/assets/29551334/acdd78e4-3a23-4ea1-965e-f262d07e1052)

Reply:
yeah, I was going to make all these versions. I just wanted to make sure I had the right content before I coped them to all the other version tabs.